wordpress-mobile/WordPress-iOS
# Stats Revamp: Updated Latest Post Summary Card

Fixes #18431. This PR updates the appearance of the Latest Post Summary insights card to match the revamp designs.

Please see also this associated WordPressKit PR: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/499. It was necessary to add an extra property to a model that's defined there.

| Before | Design | After |
|---|---|---|
| <img alt="Screenshot 2022-04-29 at 01 04 15" src="https://user-images.githubusercontent.com/4780/165866445-ed3e016b-825e-4c81-b9b2-911206405c63.png"> | <img alt="iOS-Insights-light-2" src="https://user-images.githubusercontent.com/4780/165866629-16740d93-2392-49a4-a5c3-0b05d2357f9a.png"> | ![Simulator Screen Shot - iPhone 13 Pro - 2022-04-28 at 15 54 14](https://user-images.githubusercontent.com/4780/165866620-f03de5b2-c43e-4ec1-a2bc-c15762c90889.png) |

**Known issues**

- Loading and no data states are not addressed in this PR
- Dynamic type handling requires improvement

**To test**

- First, build and run, navigate to Stats Insights for a site, add the Latest Post Summary cell and ensure the cell looks as it does on the currently shipping app.
- Now enable both of the new stats feature flags and build and run again.
- Navigate to the Latest Post Summary card, and ensure it looks as per designs. The designs show `body` font styles for some labels, but the sizes of the text looks more correct when using `subhead`, so that's what I've gone for.
- Check the cell on a number of different sites.
- Also test with posts that do and do not have featured images.

## Regression Notes
1. Potential unintended areas of impact

Existing Latest Post Summary card (with feature flag disabled).

2. What I did to test those areas of impact (or what existing automated tests I relied on)

I ensured that any relevant changes were feature flagged, and tested this card with the feature flag disabled.

3. What automated tests I added (or what prevented me from doing so)

None

PR submission checklist:
